### DISCUSSION TITLE
Type+Autosave [Oct 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4286] [Discord Threads]
> 🙋‍♂️ **alex.pro**

**Comment by o1lab** [comment]
> 🙋‍♂️ **alex.pro**  
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **true**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.98.1**  
#  
> 🙋‍♂️ **alex.pro**  
#### Should I create a ticket?  
#  
> 🙋‍♂️ **alex.pro**  
#### Looked through the issues in GH and didn't find anything.  
#  
> 🙋‍♂️ **alex.pro**  
#### https://github.com/nocodb/nocodb/issues/4146  
#  
> 🙋‍♂️ **alex.pro**  
#### Okay, I see https://github.com/nocodb/nocodb/issues/4103 🙂 sorry  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1033011218924982333)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
